plugins/package-curation-providers/api/src/main/kotlin/SimplePackageCurationProvider.kt

```diff
@@ -21,11 +21,33 @@ package org.ossreviewtoolkit.plugins.packagecurationproviders.api
 
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.plugins.api.PluginDescriptor
 
 /**
  * A [PackageCurationProvider] that provides the specified [packageCurations].
  */
-open class SimplePackageCurationProvider(val packageCurations: List<PackageCuration>) : PackageCurationProvider {
+open class SimplePackageCurationProvider(
+    override val descriptor: PluginDescriptor,
+    val packageCurations: List<PackageCuration>
+) : PackageCurationProvider {
+    constructor(packageCurations: List<PackageCuration>) : this(
+        descriptor = SimplePackageCurationProvider.descriptor,
+        packageCurations = packageCurations
+    )
+
+    companion object {
+        /**
+         * The default [PluginDescriptor] for a [SimplePackageCurationProvider]. Classes inheriting from this class
+         * should provide their own descriptor.
+         */
+        private val descriptor = PluginDescriptor(
```

Comment:
I remember @oss-review-toolkit/kotlin-devs having a discussion that we should prefer to not put `private` constants into companion objects as there is not need for scoping. So how about moving this to a top-level `private` property instead?